spec/integration/hawkular_client_spec.rb

```diff
@@ -50,24 +60,28 @@ module Hawkular::Client::RSpec
       expect { @hawkular_client.tokens_lyst_feeds }.to raise_error(NoMethodError)
     end
 
-    context 'and URIs as input', vcr: { decode_compressed_response: true } do
+    context 'and URIs as input' do
       it 'Should work with URI' do
```

Comment:
What exactly does this test? It might "work" with URI, but what exactly does this mean?